inspirehep/modules/records/receivers.py

```diff
@@ -393,6 +394,33 @@ def populate_earliest_date(sender, json, *args, **kwargs):
             json['earliest_date'] = result
 
 
+def populate_experiment_suggest(sender, json, *args, **kwargs):
+    """Populates experiment_suggest field of experiment records."""
+
+    if 'experiments.json' not in json.get('$schema'):
+        return
+
+    experiment_paths = [
```

Comment:
@michamos do these fields look fine? These are based on the original suggestions when I first attempted fixing the suggesters. :)

Reply:
yes, fields look fine. Does the order matter?